documentation/src/main/asciidoc/topics/proc_using_custom_marshallers.adoc

```diff
@@ -19,7 +19,7 @@ include::config_examples/custom_marshaller.xml[]
 [TIP]
 ====
 Custom marshaller implementations can access a configured white list via the
-link:{javadocroot}/org/infinispan/commons/marshall/Marshaller.html#initialize[initialize()] method, which is called during startup.
+link:{javadocroot}org/infinispan/commons/marshall/Marshaller.html#initialize(org.infinispan.commons.configuration.ClassWhiteList)[initialize()] method, which is called during startup.
```

Comment:
change the link to point to github? that package is private and it isn't in javadocs.

Reply:
let's remove this and the pointer to the Marshaller docs beneath it. I think we want to discourage custom marshallers if possible, right? maybe this should even just be community only.

Reply:
for Marshaller question, I'll invoke @ryanemerson  :)

Reply:
@pruivo The package is public https://github.com/infinispan/infinispan/blob/4139c97c1a34c65bbd28b1e24467a543af9aa04f/commons/all/src/main/java/org/infinispan/commons/marshall/package-info.java#L7-L6

I don't think we should dissuade the use of custom marshallers. On the client they can be very useful and have been used in the past by users.

Reply:
@pruivo I've reverted the changes and verified that it's available in javadocs. https://access.redhat.com/webassets/avalon/d/red-hat-data-grid/8.0/api/org/infinispan/commons/marshall/Marshaller.html#initialize(org.infinispan.commons.configuration.ClassWhiteList)

Reply:
no idea what link I used xD eh eh